test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -352,10 +350,7 @@ private void registerInboundFilter(ICluster<?> cluster)
                 return false;
             int fromNum = from.config().num();
             int toNum = config.num(); // since this instance is reciving the message, to will always be this instance
-            boolean permit = cluster.filters().permitInbound(fromNum, toNum, serialized);
-            if (!permit)
-                logger.info("Dropping inbound message {}", message);
```

Comment:
why are you removing this?

Reply:
this breaks most jvm-dtest... `logs()` fails as it can't find the log =(